### PR TITLE
frontend: If no claim available we don't create link for it

### PR DESCRIPTION
**Pull request by ashu8912**

### CODE REVIEW

**Comment by ashu8912** [comment]
fixes #1638

**Review by knrt10 — APPROVED** [review]
+1

File: frontend/src/components/storage/VolumeList.tsx [hunk 1]
```diff
@@ -67,6 +67,11 @@ export default function VolumeList() {
           getter: volume => {
             const claim = volume.spec.claimRef?.name;
             const claimNamespace = volume.spec.claimRef?.namespace;
+
+            if (!claim) {
+              return null;
+            }
```
Comment (joaquimrocha) [inline frontend/src/components/storage/VolumeList.tsx:73]: We should return before checking the claimNamespace, as it's unused if claim is empty/undefined anyway...

Comment (ashu8912) [reply frontend/src/components/storage/VolumeList.tsx:71]: updated

**Review by joaquimrocha — APPROVED** [review]

**Pull request merged**